src/rgw/rgw_kafka.h

```diff
@@ -0,0 +1,77 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*- 
+// vim: ts=8 sw=2 smarttab ft=cpp
+
+#pragma once
```

Comment:
is this safe to use now?  haven't seen this since...MSVC5.

Reply:
it is supported by any relevant compiler (https://en.wikipedia.org/wiki/Pragma_once#Portability).
only issue may be if we are symlinking to the same file from multiple places in our code base. i guess that for files that need that we can use the standard guards